### PR TITLE
add test class, make changes in algorithm, in test names

### PR DESCRIPTION
**Pull request by MarianBastiurea**
Salut,
Cred ca numele testelor încep sa se apropie de adevăr.

### CODE REVIEW

File: src/test/java/org/example/c01operators/calculate/CirclePerimeterAndAreaTest.java [hunk 1]
```diff
@@ -7,26 +7,10 @@
 class CirclePerimeterAndAreaTest {
 
     @Test
-    public void RadiusOne() {
+    public void givenRadiusOneAreaShouldBeNumericValueOfPiAndPerimeterShouldBeDoubleNumericValueOfPi() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c01operators/calculate/CirclePerimeterAndAreaTest.java:10]: Split this test in two tests. One for Perimeter and One for Area.

File: src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java [hunk 2]
```diff
@@ -5,29 +5,31 @@
...
+ Write a code in Java to transform numbers of days in years, weeks and days.
+
  I create an arrays with length=3. First position is for number of years
         second for number of weeks and last one for number of days
 */
 
 public class DaysToYearsWeeksAndDaysTest {
     @Test
-    public void tenLeapYear() {
+    public void given3660AsNumberOfDaysShouldBeTenLeapYear() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java:16]: This test is not correct. In 10 years there are 2 leap years.

File: src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java [hunk 3]
```diff
@@ -5,29 +5,31 @@
...
         int numberOfDays = 3660;
         int[] resultYearsWeeksDays = {10, 0, 0};
         int[] result = DaysToYearsWeeksAndDays.findYearsWeeksDays(numberOfDays);
         assertArrayEquals(result, resultYearsWeeksDays);
     }
 
     @Test
-    public void nineHundredDays() {
+    public void given900AsNumberOfDaysShouldBe2Years24WeeksAnd2Days() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java:24]: 900 is to complex input. Try with input 1 or7 or 8 or 365 or 373 or 1461(4 years of witch one is a leap year)

File: src/test/java/org/example/c01operators/calculate/EmployeeMonthlySalaryTest.java [hunk 4]
```diff
@@ -7,7 +7,7 @@
 class EmployeeMonthlySalaryTest {
 
     @Test
-    void calculateSalary() {
+    void given10AsHourlySalryAnd5WorkedHoursSalaryShouldBe50() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c01operators/calculate/EmployeeMonthlySalaryTest.java:10]: Good name! Also experiemnt with feature naming. Example "salaryIsWorkedHourstTimesHourlyRate"

File: src/test/java/org/example/c02controlflow/transform/DecimalToOctalNumberTest.java [hunk 5]
```diff
@@ -6,25 +6,9 @@
...
-        String resultOctal = "23";
-        String result = DecimalToOctalNumber.decimalToOctal(Number);
-        assertEquals(result, resultOctal);
-    }
-
-    @Test
-    public void eightyOne() {
-        int Number = 81;
-        String resultOctal = "121";
+    public void decimalNumberCouldHaveNumberBetween0And7() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c02controlflow/transform/DecimalToOctalNumberTest.java:9]: Wrong name.  Better name: octalNumberOnlyContainsNumbersFrom0To7

File: src/test/java/org/example/c02controlflow/transform/NumberToAHexadecimalNumberTest.java [hunk 6]
```diff
@@ -6,26 +6,10 @@
 
 public class NumberToAHexadecimalNumberTest {
     @Test
-    public void Ten() {
+    public void decimalNumberCouldHaveNumberBetween0to9AndCharactersBetweenAAndF() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c02controlflow/transform/NumberToAHexadecimalNumberTest.java:9]: Wrong name. Better name : hexadecimalNumberHasNumberBetween0to9AndCharactersBetweenAAndF

File: src/test/java/org/example/c01operators/calculate/CirclePerimeterAndAreaTest.java (hunk 1 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c01operators/calculate/CirclePerimeterAndAreaTest.java:10]: done

File: src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java (hunk 2 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java:16]: done

File: src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java (hunk 3 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java:24]: done

File: src/test/java/org/example/c01operators/calculate/EmployeeMonthlySalaryTest.java (hunk 4 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c01operators/calculate/EmployeeMonthlySalaryTest.java:10]: thanks

File: src/test/java/org/example/c02controlflow/transform/DecimalToOctalNumberTest.java (hunk 5 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c02controlflow/transform/DecimalToOctalNumberTest.java:9]: done

File: src/test/java/org/example/c02controlflow/transform/NumberToAHexadecimalNumberTest.java (hunk 6 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c02controlflow/transform/NumberToAHexadecimalNumberTest.java:9]: Done

File: src/test/java/org/example/c01operators/calculate/CirclePerimeterAndAreaTest.java (hunk 1 above)
Comment (andreiursudev) [reply src/test/java/org/example/c01operators/calculate/CirclePerimeterAndAreaTest.java:10]: Where?

File: src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java (hunk 2 above)
Comment (andreiursudev) [reply src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java:16]: where?

File: src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java (hunk 3 above)
Comment (andreiursudev) [reply src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java:24]: where?

File: src/test/java/org/example/c02controlflow/transform/DecimalToOctalNumberTest.java (hunk 5 above)
Comment (andreiursudev) [reply src/test/java/org/example/c02controlflow/transform/DecimalToOctalNumberTest.java:9]: where?

File: src/test/java/org/example/c02controlflow/transform/NumberToAHexadecimalNumberTest.java (hunk 6 above)
Comment (andreiursudev) [reply src/test/java/org/example/c02controlflow/transform/NumberToAHexadecimalNumberTest.java:9]: where?

**Pull request merged**